### PR TITLE
Japanese geoid GSIGEO2011 v2.1

### PR DESCRIPTION
**Pull request by hernando**
Added jp_gsi agency and GSIGEO2011 geoid grid for Japan mainland.

Validation performed with https://vldb.gsi.go.jp/sokuchi/surveycalc/geoid/calcgh/calcframe.html
The points have been choosen by hand-picking a few arbitrary locations more or less evenly distributed across Japan
The CRS is JGD2011.

        Lat     Lon      geoid h (m) reference error (mm)
        34.290  135.630  39.861751   39.8601    1.65
        36.103  140.087  40.180027   40.1817   -1.67
        43.217  143.129  30.636525   30.6389    2.37
        38.675  139.886  40.129904   40.1281    1.80
        36.344  137.654  42.895498   42.8956   -0.10
        33.179  130.063  32.303586   32.3036   -1.40
        39.801  141.322  41.886693   41.8862    0.49

The RMS is 0.9471 mm.

### CODE REVIEW

**Comment by hernando** [comment]
I'm trying to regenerate the index with GDAL 3.1.3 and PROJ 7.1.1 but it the script makes a lot of changes. The last commit was done with 3.1.4RC2?

**Comment by hernando** [comment]
@rouault I wonder whether the document with the "permission for use" from GSI should be attached.

**Comment by rouault** [comment]
> I'm trying to regenerate the index with GDAL 3.1.3 and PROJ 7.1.1 but it the script makes a lot of changes. The last commit was done with 3.1.4RC2?

I've just regenerate the index with GDAL master. The changes were due to change in forward slash escaping in GeoJSON

File: copyright_and_licenses.csv [hunk 1]
```diff
@@ -145,6 +145,8 @@ is_lmi_ISN2004_ISN2016.tif,"2017-2019, National Land Survey of Iceland",CC-BY-4.
 is_lmi_ISN93_ISN2016.tif,"2017-2019, National Land Survey of Iceland",CC-BY-4.0
 is_lmi_ISN_vel_beta.tif,"2017-2019, National Land Survey of Iceland",CC-BY-4.0
 is_lmi_README.txt,Disclaimed,Public domain
+jp_gsi_gsigeo2011_mainland.tif,Geospatial Information Authority of Japan,CC-BY-4.0,測量法に基づく国土地理院長承認(使用)R 2JHs 501
```
Comment (rouault) [inline copyright_and_licenses.csv:148]: This is a CSV file, so we can't really add a column. The extra mention is probably good enough in the readme

**Comment by rouault** [comment]
> @rouault I wonder whether the document with the "permission for use" from GSI should be attached.

@kbevers  Opinion ? We might possibly store it in the repo if it isn't too big (but make sure this is excluded from the tarball). Another option would be to send it to the OSGeo secretary so it is collected with other legal-like documents.

File: copyright_and_licenses.csv (hunk 1 above)
Comment (hernando) [reply copyright_and_licenses.csv]: Fixed

**Comment by kbevers** [comment]
> Another option would be to send it to the OSGeo secretary so it is collected with other legal-like documents.

I would say that this is the right course of action. 

FYI, I plan on releasing 7.2.0RC1 and PROJ-data 1.3RC1 on sunday or monday. It would be great if we could get this merge before that.

**Comment by hernando** [comment]
> > I'm trying to regenerate the index with GDAL 3.1.3 and PROJ 7.1.1 but it the script makes a lot of changes. The last commit was done with 3.1.4RC2?
> 
> I've just regenerate the index with GDAL master. The changes were due to change in forward slash escaping in GeoJSON

I've compiled GDA master but I get an undefined symbol error when trying to import the Python module. (demangled with c++filt it's `OGRFlatGeobufLayer::OGRFlatGeobufLayer(char const*, char const*, OGRSpatialReference*, OGRwkbGeometryType, _IO_FILE*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, bool)`)

**Comment by rouault** [comment]
@hernando Can you send the permission for use doc to secretary@osgeo.org (explaining a bit the context because Astrid won't be necessarily aware)

> I've compiled GDAL's master but I get an undefined symbol error when trying to import the Python module

Make sure to 'make clean' before rebuilding. Anyway I'll merge this PR right away, and regenerate for you since I'm setup for this

**Comment by hernando** [comment]
> @hernando Can you send the permission for use doc to [secretary@osgeo.org](mailto:secretary@osgeo.org) (explaining a bit the context because Astrid won't be necessarily aware)

Will do.

**Pull request merged**